core/src/testFixtures/java/org/lflang/tests/TestBase.java

```diff
@@ -285,8 +284,7 @@ protected static boolean isLinux() {
    */
   public static void runSingleTestAndPrintResults(
       LFTest test, Class<? extends TestBase> testClass, TestLevel level) {
-    Injector injector =
-        new LFStandaloneSetup(new LFRuntimeModule()).createInjectorAndDoEMFRegistration();
+    Injector injector = new LFStandaloneSetup().createInjectorAndDoEMFRegistration();
```

Comment:
Looks like a similar change is also needed here: https://github.com/lf-lang/lingua-franca/blob/ab5e741d0445f3200c7977a159a3a9b4a6f882f2/cli/lfc/src/test/kotlin/org/lflang/cli/LfcIssueReportingTest.kt#L103C24-L103C93